frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/details-card.tsx

```diff
@@ -70,7 +52,7 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
   const cluster = ocsData?.find((item: StorageClusterKind) => item.status.phase !== 'Ignored');
   const ocsName = getName(cluster);
 
-  const subscription = resources?.subscription as FirehoseResult;
+  const subscription = resources?.subs as FirehoseResult;
```

Comment:
is the name is changed to "subs" or it is a mistake?

Reply:
no it is not a mistake, subscriptionResource object was already defined in the "resources.ts" file and "prop" value defined there is "subs", I just imported that here. 
I did not change this "prop" value in the "resources.ts" file, because there are changes that we might be using this object at other places too.

Reply:
ack